app/src/styles.scss

```diff
@@ -36,6 +43,10 @@ body {
   height: calc(100% - 52px);
 }
 
+.mat-standard-chip {
+  padding-left: 8px !important;
```

Comment:
Will look into it, I couldn't get it to override its styling since it was coming straight from the material theme stylesheets. I will be doing a custom theme in a future iteration so that might fix it.

Reply:
All important flags have been removed